AY2021S2-CS2103-T14-1/tp
# Issue defensive coding

## What this does
- Closes #182 
- Adds assertions, `requireNonNull`, and logging for issue feature

## How to test
1. Check that the added/edited javadocs are proper
2. Check that assertion usages are proper
3. Check that logging usages are proper
